km/km_vcpu_run.c

```diff
@@ -413,7 +414,7 @@ static int hypercall(km_vcpu_t* vcpu, int* hc_ret)
          km_post_signal(vcpu, &info);
          return -1;
       }
-      if (km_hcalls_table[hc] == NULL) {
+      if (kill_unimpl_hcall != 0 && km_hcalls_table[hc] == NULL) {
```

Comment:
I did it this way because I assumed since `kill_unimpl_hcall == 0` is the default, the first expression would be false most of the time and we'd save the indirection for `km_hcalls_table[hc] == NULL`. When `kill_unimpl_hcall != 0` we are debugging and don't care about performance.

What's your reasoning for the opposite order?